src/Tizen.TV.UIControls.Forms/RecycleItemsView.cs

```diff
@@ -220,7 +220,11 @@ public DataTemplate ItemTemplate
         public object FocusedItem
         {
             get { return GetValue(FocusedItemProperty); }
-            set { SetValue(FocusedItemProperty, value); }
+            set
+            {
+                SetValue(FocusedItemProperty, value);
+                ItemFocused?.Invoke(this, new FocusedItemChangedEventArgs(FocusedItem, GetItemIndex(FocusedItem)));
```

Comment:
We already provide `RecycleItemsView.OnItemFocused()` and I think it can meet  your requirements.

Nevertheless, if you still need a `ItemFocused` event for any reason (In this case, it would be better if you explain the detailed reason), I recommend invoking it after `OnItemFocused()`. See below code.

https://github.com/Samsung/Tizen.TV.UIControls/blob/49ae40e20f49f49d29e7a432b390d9797a504ac2/src/Tizen.TV.UIControls.Forms/RecycleItemsView.cs#L723-L732

Reply:
RecycleItemsView.OnItemFocused() is protect so it can be used only when it is override.
If we provide ItemFocused makes it easier to use.
For example, if the user is using RecycleItemsView without customization, user can use events to do things like change background or title.

I modified the invoke position.